packages/renderer/src/lib/dialogs/QuickPickInput.svelte

```diff
@@ -29,58 +29,71 @@ let quickPickSelectedIndex = 0;
 let quickPickSelectedFilteredIndex = 0;
 let quickPickCanPickMany = false;
 
-let inputElement: HTMLInputElement | HTMLTextAreaElement = undefined;
-let outerDiv: HTMLDivElement = undefined;
+let inputElement: HTMLInputElement | HTMLTextAreaElement | undefined = undefined;
+let outerDiv: HTMLDivElement | undefined = undefined;
 
 const showInputCallback = async (options?: InputBoxOptions) => {
   mode = 'InputBox';
-  inputValue = options.value;
-  placeHolder = options.placeHolder;
-  title = options.title;
-  currentId = options.id;
-  if (options.prompt) {
+  inputValue = options?.value;
+  placeHolder = options?.placeHolder;
+  title = options?.title;
+  currentId = options?.id || 0;
+  if (options?.prompt) {
     prompt = `${options.prompt} (${DEFAULT_PROMPT})`;
   } else {
     prompt = DEFAULT_PROMPT;
   }
-  markdownDescription = options.markdownDescription;
-  multiline = options.multiline;
+  markdownDescription = options?.markdownDescription;
+  multiline = options?.multiline || false;
 
-  validationEnabled = options.validate;
+  validationEnabled = options?.validate || false;
   display = true;
   await tick();
 
   if (display && inputElement) {
     inputElement.focus();
-    if (options.valueSelection) {
+    if (options?.valueSelection) {
       inputElement.setSelectionRange(options.valueSelection[0], options.valueSelection[1]);
     }
   }
 };
 
 const showQuickPickCallback = async (options?: QuickPickOptions) => {
   mode = 'QuickPick';
-  placeHolder = options.placeHolder;
-  title = options.title;
-  currentId = options.id;
-  if (options.prompt) {
+  placeHolder = options?.placeHolder;
+  title = options?.title;
+  currentId = options?.id || 0;
+  if (options?.prompt) {
     prompt = options.prompt;
   }
-  quickPickItems = options.items.map(item => {
-    if (typeof item === 'string') {
-      return { value: item, description: '', detail: '', checkbox: false };
-    } else {
-      // if type is QuickPickItem use label field for the display
-      return { value: item.label, description: item.description, detail: item.detail, checkbox: false };
-    }
-  });
+  quickPickItems = options?.items
+    .map(item => {
+      if (typeof item === 'string') {
+        return { value: item, description: '', detail: '', checkbox: false };
+      } else {
+        // if type is QuickPickItem use label field for the display
+        return {
+          value: item.label || '',
+          description: item.description || '',
+          detail: item.detail || '',
+          checkbox: false,
+        };
+      }
+    })
+    .filter(item => item !== undefined) as {
+    value: any;
+    description: string;
+    detail: string;
+    checkbox: boolean;
+  }[];
+
```

Comment:
Filter is not required, because both branches of if return values and using default empty array eliminates `as` statement.
```suggestion
  quickPickItems = (options?.items || <QuickPickItem[] | string[]>[])
    .map(item => {
      if (typeof item === 'string') {
        return { value: item, description: '', detail: '', checkbox: false };
      } else {
        // if type is QuickPickItem use label field for the display
        return {
          value: item.label || '',
          description: item.description || '',
          detail: item.detail || '',
          checkbox: false,
        };
      }
    });

```